cmd/config/virt-udn-density/vm-client.yml

```diff
@@ -0,0 +1,58 @@
+apiVersion: kubevirt.io/v1
+kind: VirtualMachine
+metadata:
+  name: virt-client-{{.Iteration}}
+  labels:
+    kubevirt.io/os: cirros
+spec:
+  runStrategy: Always
+  template:
+    metadata:
+      labels:
+        kubevirt.io/os: cirros
+    spec:
+      terminationGracePeriodSeconds: 0
+      domain:
+        resources:
+          requests:
+            memory: 1Gi
+        devices:
+          disks:
+          - name: containerdisk
+            disk:
+              bus: virtio
+          - disk:
+              bus: virtio
+            name: cloudinitdisk
+          - name: emptydisk
+            disk:
+              bus: virtio
+          interfaces:
+          - name: primary-udn-net
+            binding:
+              name: passt
+      networks:
+      - name: primary-udn-net
+        pod: {}
+      volumes:
+      - name: containerdisk
+        containerDisk:
+          image: quay.io/openshift-cnv/qe-cnv-tests-fedora:40
+          imagePullPolicy: IfNotPresent
+      - name: cloudinitdisk
+        cloudInitNoCloud:
+          networkData: |
```

Comment:
I am curious if this networkData change is needed, eth0 should get dhcp by default?

Reply:
It should not change anything in 4.18.0 GA but it was needed for the older versions, let me remove it now.